webpack/assets/javascripts/react_app/components/common/emptyState/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { EmptyState as PfEmptyState, Button } from 'patternfly-react';
+
+const EmptyState = (props) => {
+  const {
+    icon = 'add-circle-o',
```

Comment:
Could you add propTypes and default props?

Reply:
And move the props destructing to `const EmptyState = ({ icon, ... }) => { ... };`

Reply:
Sure, no problem. I will check if `Katello` updated this file and add this if they don't have them too.

Reply:
Sorry but no test were found in Katello for this component... I will add them in the next days.